WalletWasabi.Tests/UnitTests/SerialBlockNotifierTests.cs

```diff
@@ -81,7 +85,7 @@ public async Task NotifyBlocksAsync()
 			var height = 0;
 			string message = string.Empty;
 
-			void OnBlockInv(object blockNotifier, Block b)
+			void OnBlockInv(object? blockNotifier, Block? b)
```

Comment:
This is not okay. This method can never accept a `null` in Block b because in that case the `b.GetHash()` would result in a NRE.
 
```suggestion
			void OnBlockInv(object? blockNotifier, Block b)
```

Reply:
You are right. Thanks.